### ISSUE TITLE
Streaming User returns no data

### ISSUE THREAD

**Issue by woakesd** [inueni/birdy#29]
Trying out the following and I'm getting no data returned from stream(). It justs blocks indefinitely.

I'm using Python 3.5.2 on Windows 10.

```
client = StreamClient(CONSUMER_KEY,
                    CONSUMER_SECRET,
                    ACCESS_TOKEN,
                    ACCESS_TOKEN_SECRET)

resource = client.userstream.user.get()
print(resource)
for data in resource.stream():
   print(data)
```

**Comment by woakesd** [comment]
I've created a pull request that fixes this.

**Comment by sect2k** [comment]
Thanks for PR, released as part of ver 0.3.
